<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetServer</class>
 <widget class="QDialog" name="SetServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SetServer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_labelServerIP">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>服务器IP：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editServerIP">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_labelServerPort">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>服务器端口：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editServerPort">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="m_pbtnConfirm">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>修改端口信息并监听</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbtnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_pbtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SetServer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pbtnConfirm</sender>
   <signal>clicked()</signal>
   <receiver>SetServer</receiver>
   <slot>ServerInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ServerInfo()</slot>
 </slots>
</ui>
